e2e/doc/code_layout.txt

```diff
@@ -0,0 +1,58 @@
+e2e
```

Comment:
The purpose of the document is trying to explain what kind of function/class that the folder contains. Would you recommend removing this file? Because the code changes may be frequent, causing this file to be forgotten or difficult to change.

Reply:
Exactly. I think we should remove this. Or consider adding a README.md in each directory to clarify the purpose.

Reply:
Ok, I will delete `code_layout.txt` first to avoid confusion.